web/src/css/tailwind.css

```diff
@@ -17,6 +17,12 @@
     overflow-wrap: anywhere;
     word-break: normal;
   }
+
+  .line-clamp-3 {
```

Comment:
use [@tailwindcss/line-clamp](https://github.com/tailwindlabs/tailwindcss-line-clamp) instead of custom class.

reference: https://tailwindcss.com/blog/multi-line-truncation-with-tailwindcss-line-clamp